tools/docker-compose/ansible/roles/sources/templates/docker-compose.yml.j2

```diff
@@ -269,6 +269,43 @@ services:
       # pg_notify will NOT work in transaction mode.
       PGBOUNCER_POOL_MODE: session
 {% endif %}
+{% if enable_otel|bool %}
+  otel:
+    image: otel/opentelemetry-collector-contrib:0.88.0
+    container_name: tools_otel_1
+    hostname: otel
+    command: ["--config=/etc/otel-collector-config.yaml", ""]
+    networks:
+      - awx
+    ports:
+      - "4317:4317"   # OTLP gRPC receiver
+      - "4318:4318"   # OTLP http receiver
+      - "55679:55679" # zpages http://localhost:55679/debug/servicez /tracez
+    volumes:
+      - "../../otel/otel-collector-config.yaml:/etc/otel-collector-config.yaml"
+      - "../../otel/awx-logs:/awx-logs/"
+    depends_on:
+      - loki
+{% endif %}
+{% if enable_loki|bool %}
+  loki:
+    image: grafana/loki:2.9.5
+    container_name: tools_loki_1
+    hostname: loki
+    ports:
+      - "3100:3100"
+    command: -config.file=/etc/loki/local-config.yaml
+    networks:
+      - awx
+    volumes:
+      - "loki_storage:/loki:rw"
+      #- "../../docker-compose/loki/volumes/index:/loki/index"
+      #- "../../docker-compose/loki/volumes/boltdb-cache:/loki/boltdb-cache"
+      - "../../loki/local-config.yaml:/etc/loki/local-config.yaml"
+    depends_on:
+      - grafana
```

Comment:
Hold on, so this requires `GRAFANA=true` if you run the other stuff? Would it be more simple to add `enable_loki` to the conditional template here for the grafana container?

Reply:
I removed the grafana dependency. It's not a hard dependency. My thought was "why turn on external logging (otel) if you aren't going to view the logs (grafana)?". Since that first thought I have added the ability to replay the log generated by otel. 

TL;DR Someone _might_ want to run otel _without_ grafana.

https://github.com/chrismeyersfsu/sosreport-elk